src/app/containers/App/App.jsx

```diff
@@ -53,6 +55,7 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
         pageType: route.pageType,
         loading: true,
         error: null,
+        errorCode: null,
```

Comment:
This should technically be `errorCode: nextErrorCode` but as we shouldnt use this in client navigation its fine :)